### PR TITLE
Fix the Sunday Mesh Morning III event date

### PR DESCRIPTION
**Pull request by kryma-foulab**

### CODE REVIEW

File: content/news/events-june-2024.md
Comment (cpatulea) [inline content/news/events-june-2024.md:1]: I think you need to add lastmod front matter (example: https://github.com/FOULAB/foulab-dot-org-hugo/commit/4bcbd629a6c7b07e47034cb4f9832a1347aebb0c)

Comment (kryma-foulab) [reply content/news/events-june-2024.md:1]: Fixed, good spot!

**Comment by cpatulea** [comment]
My mistake for closing this. I thought 'force-pushed' was to the FOULAB repo, but in fact it was to kryma's. We're trying to reopen the PR.

**Comment by cpatulea** [comment]
Superceded by https://github.com/FOULAB/foulab-dot-org-hugo/pull/20

**Pull request closed without merging**